crates/bevy_math/src/rotation2d.rs

```diff
@@ -100,18 +102,36 @@ impl Rot2 {
     };
 
     /// Creates a [`Rot2`] from a counterclockwise angle in radians.
+    ///
+    /// # Note
+    ///
+    /// The resulting rotation will always be clamped to the range (-PI, PI] by design
```

Comment:
This and the other similar comments should end in a period.

```suggestion
    /// The resulting rotation will always be clamped to the range (-PI, PI] by design.
```

Reply:
Also I noticed `as_radians` uses lowercase pi. Either that should be uppercase, or this should also be lowercase.

Reply:
Oh and also these ranges should probably be formatted as code with backticks for consistency.